### PR TITLE
User profile improvements and fixes

### PR DESCRIPTION
**Pull request by asaunier**
Related to https://github.com/c2corg/v6_ui/issues/769

### CODE REVIEW

File: c2corg_ui/templates/profile/data.html [hunk 1]
```diff
@@ -14,46 +14,48 @@
...
+            "properties": {
+              "title": ${dumps(profile["name"]) | n},
+              "lang": "${locale["lang"]}",
+              "documentId": ${profile['document_id']},
+              "module": "profiles",
+              "highlight": true
+            }
+          }]
+        };
+        detailsCtrl.documentService.setAssociations(${profile['associations']});'>
```
Comment (asaunier) [inline c2corg_ui/templates/profile/data.html]: This is the main change of this commit with the transfer of the ``app-view-details`` directive from the main template to this one (else the directive was initialized before the image associations were available. The rest of the change are simply reindents.

**Comment by asaunier** [comment]
This PR fixes and improves several part of the user profile. The main changes:
* associated images are now displayed as usual
* the same way there is an "add an outing" button in the homepage feed, I have added a "show this user's outings" button in the profile feed, which opens the outings advanced search page with the user id as criteria:
sans titre

The icon on the map is still buggy but I would prefer that @ginold fixes it in a separated PR.

**Comment by asaunier** [comment]
Anyone for this easy (I think) PR? :P

File: c2corg_ui/templates/profile/data.html [hunk 2]
```diff
@@ -14,46 +14,48 @@
...
+
+      % if profile.get('areas'):
+        <article class="location-static">
+          ${show_areas(profile)}
+        </article>
+      % endif
+
+      <div ng-init='profileFeature = {
+          "type": "FeatureCollection",
+          "properties": {},
```
Comment (ginold) [inline c2corg_ui/templates/profile/data.html]: it this needed if it's empty?

File: c2corg_ui/static/js/follow.js [hunk 3]
```diff
@@ -57,7 +57,7 @@ app.FollowController = function(appAuthentication, appApi) {
    */
   this.followed = false;
 
-  if (this.auth_.isAuthenticated()) {
+  if (this.canFollow()) {
     this.api_.isFollowing(this.docId).then(function(response) {
       this.followed = response['data']['is_following'];
```
Comment (ginold) [inline c2corg_ui/static/js/follow.js]: concerning the canFollow function - put the // comments up in the commented description section.

**Comment by ginold** [comment]
Your years of commitment and experience have paid - no issues I could detect!

File: c2corg_ui/templates/profile/data.html (hunk 2 above)
Comment (asaunier) [inline c2corg_ui/templates/profile/data.html]: That's not very clear. At least for ``Feature`` it seems required:
http://geojson.org/geojson-spec.html#feature-objects
>  A feature object must have a member with the name "properties".

I have found no indication for ``FeatureCollection``. But that doesn't hurt and it's consistent with the FeatureCollection of the other document types.

File: c2corg_ui/static/js/follow.js (hunk 3 above)
Comment (asaunier) [inline c2corg_ui/static/js/follow.js]: OK, updated at 681c31d

**Comment by asaunier** [comment]
@ginold Thanks for reviewing. I will merge when Travis passes.

**Pull request merged**